LoopKitUI/InsulinKit/InsulinDeliveryTableViewController.swift

```diff
@@ -510,29 +510,20 @@ extension DoseEntry {
         let font = UIFont.preferredFont(forTextStyle: .body)
 
         switch type {
-        case .basal, .bolus, .tempBasal:
```

Comment:
[nit] it would be great to pull this code out into a utility so it could be brought under unit test.